### PR TITLE
fix: build

### PR DESCRIPTION
**Pull request by CaptnCodr**
I hope that helps for https://github.com/fsprojects/FSharp.Configuration/pull/165

It works on my Windows 11 machine.

### CODE REVIEW

File: src/FSharp.Configuration.Runtime/FSharp.Configuration.Runtime.fsproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```
Comment (sergey-tihon) [inline src/FSharp.Configuration.Runtime/FSharp.Configuration.Runtime.fsproj]: Why you  decided to drop full framework support?

Comment (CaptnCodr) [reply src/FSharp.Configuration.Runtime/FSharp.Configuration.Runtime.fsproj]: Yea, that works with `netstandard2.0`, too. I was playing around with it.

File: paket.dependencies [hunk 2]
```diff
@@ -1,8 +1,8 @@
 source https://api.nuget.org/v3/index.json
-frameworks: net6.0, netstandard2.0
+framework: net6.0, netstandard2.0
 
 nuget YamlDotNet
-nuget FSharp.Core 4.7.2
+nuget FSharp.Core ~> 5.0.2
```
Comment (sergey-tihon) [inline paket.dependencies]: if it is work with  netstandard, you probably can keep `4.7.2` as recommended by official template -  https://github.com/fsprojects/FSharp.TypeProviders.SDK/blob/master/templates/content/basic/src/MyProvider.Runtime/MyProvider.Runtime.fsproj#L17

Comment (CaptnCodr) [reply paket.dependencies]: Yes, that works too with 4.7.2.

**Pull request merged**